projects/igniteui-angular/src/lib/calendar/calendar.component.html

```diff
@@ -4,49 +4,99 @@
 </ng-template>
 
 <ng-template let-result #defaultMonth let-obj>
-    <span tabindex="0" #monthsBtn (keydown)="onActiveViewYearKB(getViewDate(obj.index), $event, obj.index)" (click)="onActiveViewYear(getViewDate(obj.index), obj.index, $event)"
-        class="igx-calendar-picker__date">
-        {{ formattedMonth(getViewDate(obj.index)) }}
-    </span>
-    <span tabindex="0" #yearsBtn (keydown)="onActiveViewDecadeKB($event, getViewDate(obj.index), obj.index)" (click)="onActiveViewDecade(getViewDate(obj.index), obj.index)"
-        class="igx-calendar-picker__date">
-        {{ formattedYear(getViewDate(obj.index)) }}
-    </span>
+        <span class="igx-calendar__aria-off-screen" aria-live="polite">
+             {{ monthsViewNumber > 1 ? resourceStrings.first_picker + resourceStrings.of + ' ' + monthsViewNumber + ' ' + resourceStrings.starts_from + ' ' + (getViewDate(obj.index) | date: 'LLLL yyyy') : resourceStrings.igx_selected_month_is + (getViewDate(obj.index) | date: 'LLLL yyyy')}}
```

Comment:
Okay, this needs some adjusting - it's rendered for all month views (despite the text being "first of") and it's bound to month that changes for all of views when you navigate to previous/next. I realize Chromevox reads just one of those, but NVDA reads all in succession. For multiple months this only has to be rendered for the first one, right?

Also, there's no space between "first picker" and "of" and unless used elsewhere - you can make that a single locale string like `"First picker of {0} start from"`. There's no reason to translate just "of" out of context, might not even be possible in some languages.